operations/agent_monitor.go

```diff
@@ -335,8 +335,6 @@ func (m *monitor) createAgentProcess(ctx context.Context, retry util.RetryArgs)
 	if err = util.RetryWithArgs(ctx, func() (bool, error) {
 		cmd := m.jasperClient.CreateCommand(ctx).
 			Add([]string{m.shellPath, "-l", "-c", strings.Join(agentCmdArgs, " ")}).
-			SetOutputSender(level.Info, grip.GetSender()).
```

Comment:
Does the agent monitor not log anywhere? I see grip used in other places for the agent monitor.

Reply:
This makes a request to Jasper to start the agent process - the agent monitor will log regardless of these lines. I'm removing them because they don't actually do anything - since the agent is being run over gRPC, the grip senders set here are ignored because they aren't serializable and therefore ignored when starting the agent (the agent will log to splunk/a file/stdout as appropriate and these lines will have no effect on that).